Ginger/Ginger/ApplicationModelsLib/ModelParams Pages/ModelParamsPage.xaml.cs

```diff
@@ -137,10 +150,13 @@ private void InitModelParametersGrid()
             ModelParametersGrid.btnAdd.AddHandler(Button.ClickEvent, new RoutedEventHandler(AddParamsRow));
             ModelParametersGrid.AddToolbarTool("@Upgrade_16x16.png", "Upload to Global Parameters", new RoutedEventHandler(UploadToGlobalParam));
             ModelParametersGrid.AddToolbarTool("@Import_16x16.png", "Import Optional Values For Parameters", new RoutedEventHandler(ImportOptionalValuesForParameters));
+
+            ModelParametersGrid.SetbtnDeleteHandler(new RoutedEventHandler(DeleteParams_Clicked));
+            ModelParametersGrid.SetbtnClearAllHandler(new RoutedEventHandler(ClearAllParams_Clicked));
         }
         private void ImportOptionalValuesForParameters(object sender, RoutedEventArgs e)
         {            
-            WizardWindow.ShowWizard(new AddModelOptionalValuesWizard(mAAMB));
+            WizardWindow.ShowWizard(new AddModelOptionalValuesWizard((ApplicationAPIModel)mAAMB));
             ModelParametersGrid.DataSourceList = mAAMB.AppModelParameters;
```

Comment:
also AddModelOptionalValuesWizard need to accept ModelBase and not APIModel, to work with any Model

Reply:
Done